dashboard/app/controllers/projects_controller.rb

```diff
@@ -310,19 +310,37 @@ def create_new
     )
   end
 
-  # GET /projects(/script/:script_id)/level/:level_id
+  # GET /projects(/script/:script_id)/level/:level_id(/user/:user_id)
   # Given a level_id and the current user (or signed out user), get the existing project
   # or create a new project for that level and user. If a script_id is provided, get or
   # create the project for that level, script and user
```

Comment:
mind updating the comment to explain how `user_id` works here, and that it is likely different from the current user?

Reply:
Yes; done.